modules/twinkleblock.js

```diff
@@ -649,6 +649,7 @@ Twinkle.block.callback.change_action = function twinkleblockCallbackChangeAction
 
 
 		$form.find('[name=pagerestrictions]').select2({
+			theme: 'default select2-morebits',
```

Comment:
Need to do this for `$form.find('[name=namespacerestrictions]')` as well down below.